### PR TITLE
start #166841004 Setup API documentation using swagger

### PR DESCRIPTION
**Pull request by Alpha1202**
#### What does this PR do?

Api documentation using swagger

#### Description of Task to be completed?

- Install swaggerJSDoc
- Install swagger-ui-express
- Configure swagger to look for jsdoc comments in the api folder

#### How should this be manually tested?

- After cloning the repo, start the server by running `npm run start`
- Go to `http://localhost:3000/api-docs` to view the documentation

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- #166841004
- https://www.pivotaltracker.com/story/show/166841004

#### Screenshots (if appropriate)
Screenshot 2019-06-26 at 8 38 54 PM

#### Questions:
- N/A

### CODE REVIEW

**Review by chokonaira — COMMENTED** [review]
looks good

**Review by codeBlock-1984 — COMMENTED** [review]
Good job @Alpha1202.

File: .nyc_output/a3e06063-28ba-443a-902f-72187a7649ed.json [hunk 1]
```diff
@@ -0,0 +1 @@
...:{"0":{"name":"_interopRequireDefault","decl":{"start":{"line":5,"column":9},"end":{"line":5,"column":31}},"loc":{"start":{"line":5,"column":37},"end":{"line":5,"column":97}},"line":5}},"branchMap":{"0":{"loc":{"start":{"line":5,"column":46},"end":{"line":5,"column":94}},"type":"cond-expr","locations":[{"start":{"line":5,"column":70},"end":{"line":5,"column":73}},{"start":{"line":5,"column":76},"end":{"line":5,"column":94}}],"line":5},"1":{"loc":{"start":{"line":5,"column":46},"end":{"line":5,"column":67}},"type":"binary-expr","locations":[{"start":{"line":5,"column":46},"end":{"line":5,"column":49}},{"start":{"line":5,"column":53},"end":{"line":5,"column":67}}],"line":5},"2":{"loc":{"start":{"line":32,"column":8},"end":{"line":32,"column":32}},"type":"binary-expr","locations":[{"start":{"line":32,"column":8},"end":{"line":32,"column":24}},{"start":{"line":32,"column":28},"end":{"line":32,"column":32}}],"line":32}},"s":{"0":1,"1":1,"2":1,"3":1},"f":{"0":1},"b":{"0":[0,1],"1":[1,1],"2":[1,1]},"_coverageSchema":"43e27e138ebf9cfc5966b082cf9a028302ed4184","hash":"b985ed118ef3b50ce445b7e447fc9cdad58a7ecf","contentHash":"6f465eacce48f9cd40fa6bab65c24e325d97b7266e6f004310eec83a1b429775"}}
```
Comment (fob413) [inline .nyc_output/a3e06063-28ba-443a-902f-72187a7649ed.json]: Files like this shouldn't be on GitHub. Please fix this

**Review by fob413 — CHANGES_REQUESTED** [review]
Please address the little changes to this PR

**Review by fob413 — APPROVED** [review]

**Review by czarjulius — APPROVED** [review]

**Review by chokonaira — APPROVED** [review]

**Review by codeBlock-1984 — APPROVED** [review]

**Pull request merged**